documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -106,4 +110,47 @@ the thread pool size to 2-4 threads.
 - _Maximum size of single chunk in LTS: In production, it may be required to revisit the value of `writer.rollover.size.bytes.max`.
 Limiting the size of chunks in LTS poses a trade-off: if the value is low, the system will achieve fine retention granularity 
 (good) but extra metadata overhead (not so good). On the contrary, a high value in this parameter induces coarse retention 
-granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios. 
\ No newline at end of file
+granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios.
+
+## Right-Sizing Tier-1 Storage (Bookkeeper)
+
+Pravega makes use of tier-1 storage to store durably and with low latency events until they are moved to long-term storage. 
+Thus, we need guidelines to right-sizing the storage capacity required in tier-1 (i.e., Bookkeeper).
+
+To this end, there are three variables that we need to take into account:
+- _Max ledger size (`bookkeeper.ledger.size.max`)_: Maximum Ledger size (bytes) in BookKeeper. Once a Ledger reaches this size, 
+it will be closed and another one open. _Type_: `Long`. _Default_: `1073741824`. _Update-mode_: `per-server`.
+
+- _Number of Segment Containers (`pravegaservice.container.count`, `controller.container.count`)_: Number of Segment Containers in the system. 
+This value must be the same across all Segment Store instances in this cluster. This value cannot be changed dynamically. See documentation for details.
+_Type_: `Integer`. _Default_: `4`. _Update-mode_: `read-only`.
+
+- _Replication degree (`bookkeeper.write.quorum.size`)_: Write Quorum size for BookKeeper ledgers.
+_Type_: `Integer`. _Default_: `3`. _Update-mode_: `per-server`.
+
+Each Segment Container has a Durable Log (default implementation is `BookkeeperLog`). For context, a Segment
+Container multiplexes and writes all the operations of active Segments to its Durable Log. In the Bookkeeper implementation
+of the Durable Log, Segment operations are grouped into `DataFrames` and written as individual entries into a Ledger.
+If the Ledger reaches the maximum size specified in `bookkeeper.ledger.size.max`, then the current Ledger is closed and
+a new one is created (i.e., Ledger rollover). In parallel, as soon as data for a Segment Container is correctly persisted
+in long-term storage, the Durable Log can be truncated. When working with Bookkeeper, truncating a Durable Log means
+deleting the Ledgers whose last entry is lower than the last truncation point. Once these Ledgers are marked as deleted,
+Bookkeeper can delete them when running garbage collection, thus freeing up space to continue writing data.
+
+With the above context, the question to answer is what is the minimum amount of storage space that is needed
+for preventing Pravega getting rid of storage space in Bookkeeper in regular operation? We suggest provisioning storage 
```

Comment:
"for preventing Pravega from exhausting storage space on Bookkeeper for regular operations" ?

Reply:
Fixed.